config/manager/manager.yaml

```diff
@@ -18,6 +18,7 @@ spec:
     matchLabels:
       control-plane: controller-manager
       app: function-mesh-operator
+      service.istio.io/canonical-name: function-mesh-operator
```

Comment:
How about not updating the selector for the deployment? Otherwise, it requires manual intervention to upgrade existing operator deployments.